packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "invalidObject": {
+    display: "flex",
+    justifyContent: "center",
+    flexDirection: "column",
+    alignItems: "center",
+    textAlign: "center",
+    h1: { marginBottom: "30px" },
+    svg: { marginBottom: "30px" },
```

Comment:
the margins does not seem to work in the examples. Anyway, we could reimplement the component using PF EmptyState, what do you think? `NotAuthorized` component, already present in our repo is implemented in a similar way

Reply:
@fhlavac I've reimplemented it to use EmptyState